analyzer-comments/elixir/solution/module_pascal_case.md

```diff
@@ -0,0 +1,17 @@
+# module name pascal case
+
+[comment]: # (requires 2 params:)
+[comment]: # (- expected: the corrected name)
+[comment]: # (- actual: the incorrect name that the student used)
+
+Module names are always written in PascalCase in Elixir.
+
+```elixir
+# PascalCase
+
+@%{expected}
+
+# not PascalCase
+
+@%{actual}
```

Comment:
The `@`s are a leftover from module attributes, it's not a special interpolation syntax or anything :) what about:

```suggestion
defmodule %{expected} do

# not PascalCase

defmodule %{actual} do
```

Reply:
Oh, of course 😅